backend/shop/models.py

```diff
@@ -15,14 +15,19 @@ class Order(models.Model):
 
 
 class Product(models.Model):
-    image = models.ImageField(blank=True, default="", upload_to="images/products")
+    image = models.ImageField(null=True, default="", upload_to="images/products")
     title = models.CharField(max_length=255)
-    description = models.TextField(blank=True, null=True)
+    description = models.TextField(null=True)
```

Comment:
you want text based fields to be `blank=True, default=""`

Reply:
Did you think:
```
description = models.TextField(null=True, blank=True, default="")
```
or 
```
description = models.TextField(blank=True, default="")
```

Reply:
second one, do you __ever__ want your TextField to be null (`None`)?